specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/alertsSuppressionRules.json

```diff
@@ -0,0 +1,324 @@
+        "operationId": "AlertsSuppressionRules_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/AlertsSuppressionRuleName"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "OK, No Content"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AlertsSuppressionRulesList": {
+      "type": "object",
+      "description": "Suppression rules list for subscription.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AlertsSuppressionRule"
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "URI to fetch the next page."
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "AlertsSuppressionRule": {
+      "type": "object",
+      "description": "Describes the suppression rule",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/AlertsSuppressionRuleProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
+        }
+      ]
+    },
+    "AlertsSuppressionRuleProperties": {
+      "type": "object",
+      "description": "describes AlertsSuppressionRule properties",
+      "properties": {
+        "alertType": {
+          "type": "string",
+          "description": "Type of the alert to automatically suppress. For all alert types, use '*'"
+        },
+        "lastModifiedUTC": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "The last time this rule was modified"
+        },
+        "expirationDateUTC": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Expiration date of the rule"
+        },
+        "reason": {
+          "type": "string",
+          "description": "The reason for dismissing the alert"
+        },
+        "state": {
+          "type": "string",
+          "description": "Possible states of the rule",
+          "x-ms-enum": {
+            "name": "RuleState"
+          },
+          "enum": [
+            "Enabled",
+            "Disabled",
+            "Expired"
+          ]
+        },
+        "comment": {
+          "type": "string",
+          "description": "Any comment regarding the rule"
```

Comment:
Are there any restrictions on how large an input you accept on these freeform `comment`/`reason` fields?

Reply:
No need to document it in the swagger, just wanted to ask that you put some limit in the service. We have a size limit of 8Mb for an individual request/response, which means that if you allow users to put huge amounts of data into these fields, your APIs may stop working.

The limit doesn't need to be too restrictive, just want to ensure we don't allow >100kb of data for example - as you potentially need to also return multiple resources in one response payload on your list API.

Reply:
I'll add a limit of 300 characters